src/XMakeTasks/AssemblyDependency/AssemblyInformation.cs

```diff
@@ -51,6 +58,7 @@ internal AssemblyInformation(string sourceFile)
             ErrorUtilities.VerifyThrowArgumentNull(sourceFile, "sourceFile");
             _sourceFile = sourceFile;
 
+#if FEATURE_ASSEMBLY_LOADFROM
             if (NativeMethodsShared.IsWindows)
```

Comment:
@rainersigwald

> Do we need this check any more, or is it superseded by `FEATURE_ASSEMBLY_LOADFROM`?

I think we still need it for Mono, which supports Assembly.LoadFrom.